src/v/kafka/server/handlers/describe_configs.cc

```diff
@@ -327,7 +384,7 @@ ss::future<response_ptr> describe_configs_handler::handle(
         }
 
         case config_resource_type::broker:
-            report_broker_config(result);
+            report_broker_config(result, request.data.include_synonyms);
```

Comment:
why not simply always return the include_synonyms why hwave an additional branch?

Reply:
this is part of the Kafka protocol. It has a field that allows requester to either include synonyms or not, we have to support this.

Reply:
ACK. thank you.